PayPal/build.gradle

```diff
@@ -35,7 +35,7 @@ dependencies {
     api project(':BraintreeCore')
 
     implementation deps.appCompat
-    implementation project(':BraintreeDataCollector')
+    implementation project(':PayPalDataCollector')
```

Comment:
is this a breaking change?

Reply:
Fortunately it should be good in a minor release! If it was marked as `api` then it would definitely break but these modules were including `:BraintreeDataCollector` only to use the package private `PayPalDataCollector` class which up until now was invisible to merchants.

Reply:
Here in [PayPalInternalClient](https://github.com/braintree/braintree_android/blob/0059a909b99ae492eb99542aaa5a523c5a740f2c/PayPal/src/main/java/com/braintreepayments/api/PayPalInternalClient.java#L23) and [LocalPaymenetClient](https://github.com/braintree/braintree_android/blob/0059a909b99ae492eb99542aaa5a523c5a740f2c/LocalPayment/src/main/java/com/braintreepayments/api/LocalPaymentClient.java#L25), `PayPalDataCollector` is the only class referenced from the `:BraintreeDataCollector` module.

At the moment, we're pulling in a public module to reference a hidden internal class. This is the first time we're putting PayPal specific data collection (without Kount) in GA.